web/src/engine/websites/TruyenQQ.ts

```diff
@@ -0,0 +1,54 @@
+import { Tags } from '../Tags';
+import icon from './TruyenQQ.webp';
+import type { Chapter} from '../providers/MangaPlugin';
+import { type MangaPlugin, DecoratableMangaScraper, type Manga, type Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { Priority, TaskPool } from '../taskpool/TaskPool';
+import { RateLimit } from '../taskpool/RateLimit';
+import { Numeric } from '../SettingsManager';
+import { WebsiteResourceKey as R } from '../../i18n/ILocale';
+
+function PageExtractor(element: HTMLElement): string {
+    return element.dataset.original;
+}
+
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    private readonly interactionTaskPool = new TaskPool(1, new RateLimit(30, 60));
+
+    public constructor() {
+        super('truyenqq', 'TruyenQQ', 'https://truyenqqviet.com', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Language.Vietnamese, Tags.Source.Aggregator);
+        this.Settings.throttle = new Numeric('throttle.interactive', R.Plugin_Settings_ThrottlingInteraction, R.Plugin_Settings_ThrottlingInteractionInfo, 30, 1, 60);
+        (this.Settings.throttle as Numeric).Subscribe(value => this.interactionTaskPool.RateLimit = new RateLimit(value, 60));
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override ValidateMangaURL(url: string): boolean {
+        return new RegExp(`^${this.URI.origin}/truyen-tranh/[^/]+$`).test(url);
+    }
+
+    public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga> {
+        return this.interactionTaskPool.Add(async () => Common.FetchMangaCSS.call(this, provider, url, 'h1[itemprop="name"]'), Priority.Normal);
+    }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const mangaList = [];
+        for (let page = 1, run = true; run; page += 1) {
+            const mangas = await this.interactionTaskPool.Add(async () => Common.FetchMangasSinglePageCSS.call(this, provider, `/truyen-moi-cap-nhat/trang-${page}.html`, 'ul.list_grid li h3 a'), Priority.Normal);
```

Comment:
Is this normal priority by intention instead of low priority? Users won't be able to load chapters/pages for reading while manga list is acquired.

Reply:
I pasted this code from elsewhere, also for manga listing, and it was normal. Ofc i can set ti to low :D